commands/addons/create.js

```diff
@@ -2,12 +2,13 @@
 
 const cli = require('heroku-cli-util')
 const co = require('co')
+let waitForAddonProvisioning = require('../../lib/addons_wait')
 
 function parseConfig (args) {
   let config = {}
   while (args.length > 0) {
     let key = args.shift()
-    if (!key.startsWith('--')) throw new Error(`Unexpected argument ${key}`)
+    if (!key.startsWith('--')) { throw new Error(`Unexpected argument ${key}`) }
```

Comment:
Right, I don't have a great answer for that. @garyposter and I have talked about it a few times but we've yet to come up with a better solution. I just don't like it.